client/payment-details/summary/index.tsx

```diff
@@ -113,10 +113,10 @@ const PaymentDetailsSummary = ( {
 		featureFlags: { isAuthAndCaptureEnabled },
 	} = useContext( WCPaySettingsContext );
 
-	// We should only fetch the authorization data if the payment is marked for manual capture
+	// We should only fetch the authorization data if the payment is marked for manual capture and it is not already captured.
 	const shouldFetchAuthorization =
-		charge.amount !== charge.amount_captured &&
-		charge.amount_refunded === 0 &&
+		'captured' in charge && // captured key is only present for manual capture payments. See https://stripe.com/docs/api/charges/object#charge_object-captured
```

Comment:
Is there an alternative syntax for this? IDK, 'typeof charge.captured !== undefined' or something in that direction?

Reply:
For the record, I was thinking that the captured key will only be present in the object for manually captured payments, but I was wrong. Please check the comment below.

That's why I was using the "in" operator, but as I said this is pointless. Anyway thanks for the hint, I'll think twice before using it again, as it seems not to be readable.